src/pcf857x.ts

```diff
@@ -230,9 +239,14 @@ export abstract class PCF857x<PinNumber extends PCF8574.PinNumber | PCF8575.PinN
       // decrease the use count of the GPIO and unexport it if not used anymore
       this._gpio['pcf857xUseCount']--;
       if (this._gpio['pcf857xUseCount'] === 0) {
+        // error-proof in case somehow a negative value for gpioPin was somehow allowed during enableInterrupt() call
+        if (this._gpioPin >= 0) {
+          // nullify the registered gpio from our allInstancesUsedGpios array as count is 0 and it is being unexported
+          PCF857x._allInstancesUsedGpios[this._gpioPin] = null;
```

Comment:
Should be
```ts
delete PCF857x._allInstancesUsedGpios[this._gpioPin];
```
to delete the key from the object. To store the `null` value is not necessary here.

Reply:
It's an object. So the key can safely deleted without affecting the other keys.

Reply:
yeah, I thought was an array for some reason.  PR on the way.